packages/interactivity/docs/2-api-reference.md

```diff
@@ -363,20 +365,88 @@ store( "myPlugin", {
 </details>
 <br/>
 
-The `wp-on` directive is executed each time the associated event is triggered. 
+The `wp-on` directive is executed each time the associated event is triggered.
 
 The callback passed as the reference receives [the event](https://developer.mozilla.org/en-US/docs/Web/API/Event) (`event`), and the returned value by this callback is ignored.
 
-#### `wp-watch` 
+#### `wp-on-window`
+
+It allows to attach global window events like `resize`, `copy`, `focus` and then execute a defined callback when those happen.
+
+[List of supported window events.](https://developer.mozilla.org/en-US/docs/Web/API/Window#events)
+
+> The syntax of this directive is `data-wp-on--[window-event]` (like `data-wp-on-window--resize`
```

Comment:
```suggestion
> The syntax of this directive is `data-wp-on-window--[window-event]` (like `data-wp-on-window--resize`
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57931/commits/0e632bfe8f0f254364ee782af116805009d725bd